Gemfile

```diff
@@ -123,6 +122,7 @@ group :development do
 end
 
 group :development, :test do
+  gem "bullet", "~> 6.0" # help to kill N+1 queries and unused eager loading
```

Comment:
Bullet can be useful in testing mode as well, if requested it will raise an exception each time it encounters a N+1 query.